examples/python/spear_qcp/scenario.txt

```diff
@@ -2,6 +2,7 @@ algo = python -u ./spear_qcp/target_algorithm/scripts/SATCSSCWrapper.py --mem-li
 paramfile = ./spear_qcp/target_algorithm/spear-python/spear-params-mixed.pcs
 execdir = .
 deterministic = 0
+limit_resources = 1
```

Comment:
why do we limit resources here?

Reply:
Because `limit_resources` was default before. Otherwise, test will fail.